content/definition.it.md

```diff
@@ -1,14 +1,44 @@
 +++
-title = "La definizione di Sorgente Etica"
+title = "La definizione di Sorgente Etica (SE)"
 template = "definition"
+heading="La definizione di Sorgente Etica"
 +++
 
-Un Software è considerato Sorgente Etica (_Ethical Source_) se rispetta i seguenti criteri:
+Un Software è considerato **Open-Source Etico** (EOS) se rispetta *tutti* i seguenti criteri:
 
-1. E' liberamente distribuito con il codice sorgente e puó essere utilizzato o combinato con altri software senza alcuna licenza o prezzo.
-2. E' sviluppato pubblicamente e accetta contributi dalla comunitá.
-3. La sua community è governata da un codice di condotta che è consistente ed imposta.
-4. I suoi creatori hanno il diritto di proibire il suo utilizzo a individui oppure organizzazioni che violano i diritti umani o hanno comporamenti considerati non etici.
-5. I suoi creatori hanno il diritto di sollecitare compensazioni ragionevoli e volontarie dalla comunitá o istituzioni che beneficiano dall'utilizzo del software.
+1. **È di comune beneficio.** La licenza con la quale il software è distribuito non ne proibisce la modifica, la derivazione e il linkaggio o la compilazione con altro software (se non in virtù di una disposizione copyleft), nello spirito generale della distribuzione open source.
```

Comment:
I do not think "linkaggio" is an Italian world. Wikipedia keep it as `Linking`, so I would do the same here: https://it.wikipedia.org/wiki/Linking.

Reply:
`la derivazione` comes from the concept that you can create a new thing Y starting from X (https://it.wikipedia.org/wiki/Derivazione_(linguistica)). I found this word used in many textbooks, ndependently from the industry. But I'll switch to `lavori derivati`, which is less ambiguous